packages/openbridge-webcomponents/src/components/app-button/app-button.css

```
@@ -74,6 +74,8 @@
   }
 
   &.integration {
+    @mixin style style=integration-flat;
```

Stylelint (17.14.0)

[error] Expected mixin name "style style=integration-flat" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)